test/paper-textarea.html

```diff
@@ -194,39 +194,6 @@
         assert(!input.focused, 'input is blurred');
       });
     });
-
-    suite('a11y', function() {
```

Comment:
Because the accessibility function has moved to PolymerElements/iron-autogrow-textarea so also have the test.